pkg/apis/kubermatic/v1/preset.go

```diff
@@ -49,22 +49,38 @@ type Preset struct {
 
 // Presets specifies default presets for supported providers.
 type PresetSpec struct {
-	Digitalocean        *Digitalocean        `json:"digitalocean,omitempty"`
-	Hetzner             *Hetzner             `json:"hetzner,omitempty"`
-	Azure               *Azure               `json:"azure,omitempty"`
-	VSphere             *VSphere             `json:"vsphere,omitempty"`
-	AWS                 *AWS                 `json:"aws,omitempty"`
-	Openstack           *Openstack           `json:"openstack,omitempty"`
-	Packet              *Packet              `json:"packet,omitempty"`
-	GCP                 *GCP                 `json:"gcp,omitempty"`
-	Kubevirt            *Kubevirt            `json:"kubevirt,omitempty"`
-	Alibaba             *Alibaba             `json:"alibaba,omitempty"`
-	Anexia              *Anexia              `json:"anexia,omitempty"`
-	Nutanix             *Nutanix             `json:"nutanix,omitempty"`
+	// Access data for Digital Ocean.
```

Comment:
`DigitalOcean` is the correct name, without a space in between.

Reply:
My bad. will fix it.!